test/e2e/charts/flow-visibility/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Settings for ClickHouse
+secureConnection:
+  enable: false
+  # -- Common name to use in the certificate.
+  commonName: "clickhouse-clickhouse.flow-visibility.svc"
+  # -- IP addresses to use in the certificate.
+  ipAddresses: []
+  # -- DNS names to use in the certificate.
+  dnsNames: []
+  # -- Number of days for which the certificate will be valid. There is no automatic rotation with
+  # this method.
```

Comment:
Remove the second sentence. It is not really applicable to this chart (and there is only a single supported "method").

Reply:
Updated. Thanks.